From fb51714f903f73a203dd9057cc2d936f3b501f86 Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Fri, 27 May 2016 05:18:13 +0000
Subject: [PATCH 86/86] misc: lsi-smmon: change the include file to right
 path.

It's because that the include file ncr.h has moved to include/linux
directory and renamed as lsi-ncr.h.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/misc/lsi-smmon.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/misc/lsi-smmon.c b/drivers/misc/lsi-smmon.c
index dd02647..acebe53 100644
--- a/drivers/misc/lsi-smmon.c
+++ b/drivers/misc/lsi-smmon.c
@@ -21,7 +21,7 @@
 #include <linux/of_address.h>
 #include <linux/reboot.h>
 
-#include <mach/ncr.h>
+#include <linux/lsi-ncr.h>
 
 #define APB2_SER3_PHY_ADDR        0x002010030000ULL
 #define APB2_SER3_PHY_SIZE   0x1000
-- 
1.7.5.4

